content/beginner/190_secrets/create-a-secret.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Create a Secret"
+date: 2019-04-09T00:00:00-03:00
+weight: 10
+draft: false
+---
+
+
+#### Encrypt Your Secret
+
+Create a namespace for this exercise:
+
+```bash
+kubectl create ns secretslab
+```
+Output: 
+{{< output >}}
+namespace/secretslab created
+{{< /output >}}
+
+Create a text file contianing your secret:
+```bash
+echo -n "am i safe?" > ./test-creds
+```
+
+
+Create your secret
+```bash
+kubectl create secret \
+        generic test-creds \
+        --from-file=test-creds=./test-creds \
+        --namespace secretslab
+```
+Output: 
+{{< output >}}
+secret/test-creds created
+{{< /output >}}
+
+Retrieve the secret via the CLI:
+```bash
+kubectl get secret test-creds \
+  -o jsonpath="{.data.test-creds}" \
+  --namespace secretslab | \
+  base64 --decode
+```
```

Comment:
This fails as well. Probably needs reformatting. Works on a single line with 'kubectl get secret test-creds -n secretslab -o jsonpath="{.data.test-cree64 --decode'